### PR TITLE
Preparing for the next mentorship round!

### PR DESCRIPTION
**Pull request by realvictorprm**
Okay this are my proposed changes, any comments on this?
cc @selketjah, @ReedCopsey,

### CODE REVIEW

**Comment by realvictorprm** [comment]
updated, please review again

File: mentorship/about.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The program is free-of-charge. The mentors are volunteering their time to benefi
 
 ### What do I need to do next?
 
-Please register at [this page](register.html).
+Unfortunately you need to wait until beginning of August, then further information will be available [here](register.html). Any updates will be on twitter ([@fsharpedu](https://twitter.com/fsharpedu)) too.
```
Comment (ReedCopsey) [inline mentorship/about.md]: I'd used @fsharporg twitter.  Also "wait until the beginning of"

Comment (realvictorprm) [inline mentorship/about.md]: Change to @fsharporg because @fsharpedu is abandoned.

File: mentorship/register.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ layout: default
 title: Registering for the F# Mentorship program | The F# Software Foundation
 headline: Registering for the F# Mentorship program
 ---
-
-**Registration is currently closed. We hope to do another round in the future.**
+**Another mentorship round is planned for the beginning of September, registration will be opened in the beginning of August!**
```
Comment (ReedCopsey) [inline mentorship/register.md]: I'd make this not bold, and also mention following @fsharporg twitter handle here, too.

Comment (realvictorprm) [inline mentorship/register.md]: 👍

**Comment by ReedCopsey** [comment]
Thanks @realvictorprm !

**Comment by realvictorprm** [comment]
Uh nice 💯

**Pull request merged**